hera_qm/tests/test_xrfi.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 # Copyright (c) 2018 the HERA Project
 # Licensed under the MIT License
-
 from __future__ import print_function, division, absolute_import
+import hera_qm.tests as qmtest
```

Comment:
err, this import already shows up below (line 14). Also, it's not used. So let's chop this line and 14.

Reply:
![image](https://user-images.githubusercontent.com/8646829/45564156-6e9d9080-b804-11e8-9438-d6258e1404c2.png)